.travis.yml

```diff
@@ -74,40 +78,29 @@ install:
 
 jobs:
   include:
-    - stage: lint
-      python: 3.7
+    - stage: Lint
+      python: 3.6
```

Comment:
Maybe we should leave the linter at python 3.7 here? It looks like it was bumped up at some point in 4.0's history but not in master.

Reply:
OK!  I'll let this run and then update.

Reply:
On second thought, maybe not since we're testing w/ py 3.6 and 3.8, so why use an odd numbered version?

Reply:
So maybe bump the linter up to 3.8!